Tempita 0.5.2 cannot be installed from PyPI

We have a dependency on [Tempita](https://pypi.org/project/Tempita/) (through sqlalchemy-migrate), which is abandonware and uses the `use_2to3=True` option of `setup()` in its `setup.py`
The new Setuptools v58.0.0 removed support for 2to3 during builds: https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v5800
As a result Tempita is installed at even older version and importing it raises a SyntaxError:
  ```
  from tempita import Template as TempitaTemplate
  E     File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/tempita/__init__.py", line 295
  E       except SyntaxError, e:
  E                         ^
  E   SyntaxError: invalid syntax
  ```

_Originally posted by @nsoranzo in https://github.com/galaxyproject/galaxy/issues/12408#issuecomment-913577724_

--- a/lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt
+++ b/lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt
@@ -11,7 +11,7 @@ adal==1.2.5
 amqp==5.0.2; python_version >= '3.6'
 appdirs==1.4.4
 argcomplete==1.12.2
-attmap==0.12.11
+attmap==0.13.0
 attrs==20.3.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 babel==2.9.0
 bagit==1.7.0

--- a/packages/data/requirements.txt
+++ b/packages/data/requirements.txt
@@ -13,4 +13,5 @@ social-auth-core[openidconnect]==3.3.0
 SQLAlchemy>=1.3.0,<1.4.0  # https://github.com/kvesteri/sqlalchemy-utils/issues/474
 sqlalchemy-migrate
 sqlalchemy-utils
+tempita @ git+https://git@github.com/nsoranzo/tempita@main#egg=tempita  # Required by sqlalchemy-migrate, see https://github.com/galaxyproject/galaxy/issues/12412
 WebOb

